raiden/accounts.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+import json
+import getpass
+
+from pyethapp.accounts import Account
+
+
+def find_datadir():
+    home = os.path.expanduser('~')
+    if home == '~':  # Could not expand user path
+        return None
+    datadir = None
+
+    if sys.platform == 'darwin':
+        datadir = os.path.join(home, 'Library', 'Ethereum')
+    # NOTE: Not really sure about cygwin here
+    elif sys.platform == 'win32' or sys.platform == 'cygwin':
+        datadir = os.path.join(home, 'AppData', 'Roaming', 'Ethereum')
+    elif os.name == 'posix':
+        datadir = os.path.join(home, '.ethereum')
+    else:
+        raise RuntimeError('Unsupported Operating System')
+
+    if os.path.isdir(datadir) is False:
+        return None
+    return datadir
+
+
+def find_keystoredir():
+    datadir = find_datadir()
+    if datadir is None:
+        # can't find a data directory in the system
+        return None
+    keystore_path = os.path.join(datadir, 'keystore')
+    if not os.path.exists(keystore_path):
+        # can't find a keystore under the found data directory
+        return None
+    return keystore_path
+
+
+class AccountManager(object):
+
+    def __init__(self, keystore_path=None):
+        self.keystore_path = keystore_path
+        self.accounts = {}
+        if self.keystore_path is None:
+            self.keystore_path = find_keystoredir()
+        if self.keystore_path is not None:
+
+            for f in os.listdir(self.keystore_path):
+                fullpath = os.path.join(self.keystore_path, f)
+                if os.path.isfile(fullpath):
+                    with open(fullpath) as data_file:
```

Comment:
context managers are one place were short names are more "idiomatic" so it's common to see `... as f:`